helpme/models.py

```diff
@@ -37,6 +37,10 @@ class VisibilityChoices(models.IntegerChoices):
     DEVELOPERS = 15, _("Developers")
     SUPERVISORS = 20, _("Supervisors")
 
+class CommentTypeChoices(models.IntegerChoices):
+    MESSAGE = 0, _('Message')
+    HISTORY = 1, _('History')
```

Comment:
Would EVENT be more descriptive as to what it is?  This is up to you if you want to change it.